### DISCUSSION TITLE
Question related to cloudformation.schema.json

### DISCUSSION THREAD

**Discussion by ikanni** [aws/serverless-application-model#3355] [Questions] [answered]
Does anyone know the correct github repo to ask question related to [cloudformation.schema.json](https://github.com/aws/serverless-application-model/blob/main/schema_source/cloudformation.schema.json)?

**Comment by GavinZZ** [comment, marked as answer]
I know there is a github open source repo, https://github.com/aws-cloudformation/cloudformation-template-schema. Maybe try this?

**Reply by ikanni** [reply]
Thank you!

**Comment by GavinZZ** [comment]
Closing this discussion as it's technically unrelated to SAM.
